.github/workflows/charm_release_edge.yml

```diff
@@ -0,0 +1,24 @@
+name: Release testflinger-k8s charm to latest/edge
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Release to Charmhub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Upload charm to charmhub
+        uses: canonical/charming-actions/upload-charm@2.3.0
+        with:
+          charm-path: charm
+          credentials: "${{ secrets.CHARMHUB_TOKEN }}"
```

Comment:
Just for reference, when implementing test observer's action. What's charmhub token?

Reply:
You would need to upload a new one under the test_observer project called "CHARMHUB_TOKEN". I just created one with my own account for now by using `charmcraft login --export /tmp/charmcraft.auth --charm=testflinger-k8s --permission=package-manage --channel=edge --ttl=2592000`
However, I'm working at figuring out who to pester so that we can get the ownership changed on those charms. So hopefully we can use the bot account soon!

Reply:
oh yeah forgot about `charmcraft login --export`. But yeah moving ownership is an issue